cli/lib/tasks/install.js

```diff
@@ -55,12 +55,16 @@ const displayCompletionMsg = () => {
 
   logger.log()
   logger.log(
-    'You can now open Cypress by running:',
-    chalk.cyan(path.join('node_modules', '.bin', 'cypress'), 'open'),
+    'You can now open Cypress by running one of the following, depending on your package manager:',
   )
 
   logger.log()
-  logger.log(chalk.grey('https://on.cypress.io/installing-cypress'))
+  logger.log(chalk.cyan('npx cypress open'))
+  logger.log(chalk.cyan('yarn cypress open'))
+  logger.log(chalk.cyan('pnpm cypress open'))
```

Comment:
Maybe it would look better with a list too? Sorry for more changes. It looks like a wall of text when see the examples. There is some convention of this in other places: https://github.com/cypress-io/cypress/blob/develop/cli/lib/tasks/install.js#L49

```suggestion
  logger.log(chalk.cyan('- npx cypress open'))
  logger.log(chalk.cyan('- yarn cypress open'))
  logger.log(chalk.cyan('- pnpm cypress open'))
```

Reply:
I agree again with your formatting suggestion. Addressed in https://github.com/cypress-io/cypress/pull/29957/commits/11e5f0d4ba53fc4ae622c395c15efd6ad2b1b4f5.

I also updated with a screenshot to the original posting which also shows the colors rendered.